docs/UserGuide.md

```diff
@@ -1226,6 +1233,8 @@ Format: `editcomment STUDENT_INDEX as/ASSIGNMENT c/COMMENT`
 * The student index refers to the index number shown in the displayed student list.
 * The student index **must be a positive integer** 1, 2, 3, …​ that is within the range of the student list.
 * The name of the assignment must exist in F.A.K.E.J.A.R.V.I.S.
+* The comment **must not be empty**
+* The comment **must be within 200 characters**.
```

Comment:
hmm, im not sure if the code checks for char > 200.
If we leave this in, and no error pops out, it might be considered a bug

Reply:
Is part of the code, so prob shld include bah